src/registrar/tests/test_views.py

```diff
@@ -144,6 +144,42 @@ def test_application_form_empty_submit(self):
         result = page.form.submit()
         self.assertIn("What kind of U.S.-based government organization do you represent?", result)
 
+    def test_application_multiple_applications_exist(self):
+        """Test that an info message appears when user has multiple applications already"""
+        # create and submit an application
+        contact = Contact.objects.create()
```

Comment:
**question (non-blocking)**: There is a `create_full_dummy_domain_application` that might streamline this first section of the test?

Reply:
thanks @neilmb I streamlined the first section of the test at your suggestion